shuup/core/models/_media.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from filer.fields.file import FilerFileField
+from filer.fields.folder import FilerFolderField
+
+
+@python_2_unicode_compatible
+class MediaFile(models.Model):
+    file = FilerFileField(related_name="media_file", verbose_name=_("file"), on_delete=models.CASCADE)
+    shops = models.ManyToManyField(
+        "shuup.Shop", related_name="+", verbose_name=_('shops'),
```

Comment:
I like very much using related_names. When you add `+` that will be hidden ans there is no way of selecting all shop media folders through a related queryset like `ShopProducts.objects.filter(shops__media_files__XXX)`. I know you can't see any usage right now but maybe we shouldn't kill this possibility. What do you think?